src/utils/utils.ts

```diff
@@ -61,3 +61,60 @@ export function settingsListToTree(list: Setting[]): Tree {
   });
   return root;
 }
+
+
+export function deleteLocalKey(key : string, id : string) {
```

Comment:
maybe we should put into on class? this would dedupe all `Local` strings in the function